harbor/README.md

```diff
@@ -0,0 +1,13 @@
+# Harbor Configuration 
+
+Below is table that highlights the required fields for the Harbor data values file. 
+
+|  Property	                                                 | Value	              | Description    |                                                                                         
+| ---------------------------------------------------------- | -------------------- | -------------- |                                                                                        
+| hostname	                                                 | FQDN	                | The FQDN that you have designated to access the Harbor UI and for referencing the registry in client applications.|
```

Comment:
Might be good to add a note here: The domain should be configured in an external DNS server such that it resolves to the Envoy Service IP created by Contour.